edx_proctoring/static/proctoring/spec/proctored_exam_attempt_spec.js

```diff
@@ -59,7 +59,7 @@ describe('ProctoredExamAttemptView', function () {
         '<div class="search-attempts">' +
         '<input type="text" id="search_attempt_id" placeholder="e.g johndoe or john.doe@gmail.com"' +
         '<% if (inSearchMode) { %> value="<%= searchText %>" <%} %>' +
-        '/> <span class="search"><i class="fa fa-search"></i></span> <span class="clear-search"><i class="fa fa-remove"></i></i></span>' +
+        '/> <span class="search"><span class="fa fa-search" aria-hidden="true"></span></span> <span class="clear-search"><span class="fa fa-remove" aria-hidden="true"></span></i></span>' +
```

Comment:
Do you need to add the ``icon`` class too? This is the recommendation from the pattern library doc:

    <span class="icon fa fa-info-circle" aria-hidden="true"></span>